game/game/packets/system_message.py

```diff
@@ -0,0 +1,36 @@
+import typing
+from dataclasses import dataclass, field
+
+from common.ctype import ctype
+from common.misc import extend_bytearray
+from game.models.structures.system_message import SystemMessage, ZoneName
+from game.packets.base import GameServerPacket
+
+
+@dataclass(kw_only=True)
+class SystemMessagePacket(GameServerPacket):
+    type: ctype.int8 = field(default=100, init=False)
+    message: SystemMessage
+
+    def encode(self, session):
+        encoded = bytearray(self.type)
+        encoded.extend(bytes(ctype.int32(self.message.type)))
+        encoded.extend(bytes(ctype.int32(len(self.message.values))))
```

Comment:
please use `extend_bytearray` function to extend bytearray

Reply:
it was done in [this commit](https://github.com/Yurzs/L2py/pull/23/commits/ca82c2bab8cbc72cb5d33d3bc11de984b2b6dd27)